packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.test.tsx

```diff
@@ -32,6 +32,23 @@ describe('ColorPickerPopover', () => {
       await userEvent.click(colorSwatchWrapper[0]);
       expect(color).toHaveStyle('box-shadow: inset 0 0 0 2px #73BF69,inset 0 0 0 4px #000000');
     });
+
+    it('should maintain the same height when switching tabs', async () => {
```

Comment:
This test is very specific about the DOM order and will break once that's changed. Tbh, I don't think we need a test for the case of the picker height.